daemon/src/model/cfd.rs

```diff
@@ -657,6 +662,16 @@ impl Cfd {
             return Err(RolloverError::WrongRole);
         }
 
+        // TODO: Adjust for total paid fees
+        let margin_minus_total_fees = self.margin(self.position);
+
+        // TODO: Calculate the time from the last rollover, don't just assume
+        // it's up-to-date
+        let hours_to_charge = 1;
+
+        let funding_params =
+            FundingParams::new(margin_minus_total_fees, self.funding_rate, hours_to_charge)?;
```

Comment:
I don't really understand the need for the `FundingParams` struct.

It should be a stateless calculation to figure out the funding fee upon rollover. Why do we need to store stuff in a struct for that?

Reply:
hm, perhaps it could be a free function after all! will give it a shot :)